app/api/resources/user.py

```diff
@@ -120,6 +121,9 @@ def put(cls):
         """
         user_id = get_jwt_identity()
         data = request.json
+        is_valid = validate_new_password(data['new_password'])
```

Comment:
instead of receiving the data[`new_password`] you could receive data and do all the verifications because that will give an exception if `new_password` isn't there @sammy1997